src/createReducer.ts

```diff
@@ -60,6 +67,15 @@ export function createReducer<Present, Actions extends {}>(
       }
     }
 
+    const isTrackableAction = typeof options?.track === 'function' ? options.track(action) : trackAll()
```

Comment:
Do you think the way I manage optional value is okay for you? 

The alternative was `options?.track?.(action) ?? trackAll()`, but this way we'll pass the control to the `trackAll` function if `track` returns `undefined` (for example, if people use JS). I'd rather consider it as `false`. 

Feel free to let me know if you have any doubts. Maybe I'm overthinking the solution here.

Reply:
After consideration, I decided that a better API would be to consider only explicit `false` as a way to skip the action. In that approach `undefined` means — use the default, which is `true`